Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/POMScreenShotWizardPage.xaml.cs

```diff
@@ -57,9 +57,8 @@ public void WizardEvent(WizardEventArgs WizardEventArgs)
 
         public void ShowScreenShot()
         {
-            mWizard.IWindowExplorerDriver.UnHighLightElements();
-            mWizard.ScreenShot = ((IVisualTestingDriver)mWizard.Agent.Driver).GetScreenShot();
             mScreenshotPage = new ScreenShotViewPage(mWizard.POM.Name, mWizard.ScreenShot);
+            mScreenshotPage.EnlargeButton.Visibility = System.Windows.Visibility.Hidden;
             MainFrame.Content = mScreenshotPage;
         }
```

Comment:
Screen shot was merged into "Details" wizard page so "POMScreenShotWizardPage" not supposed to be deleted?

Reply:
Yes, Deleted it.